src/plugins/stake-plugins/generic/pluginInfo/ethereumKilnPool.ts

```diff
@@ -15,6 +15,7 @@ const kilnPolicyConfig: Array<StakePolicyConfig<EthereumPooledKilnAdapterConfig>
     adapterConfig: {
       exitQueueAddress: '0x8979117a69DfA7F4D4E3c7B59197ff03f4A2CeAF',
       type: 'ethereum-pooled-kiln',
+      accountId: ENV.KILN_TESTNET_ACCOUNT_ID,
```

Comment:
I don't understand why this was added. It seems like implementation details for ADA staking are becoming unnecessary implementation details for ETH.

Reply:
It's because it's a requirement for the shared KilnApi lib. Shall I make it optional, then I'd need to add assertions to each KilnApi method that requires it... I could do that if you think that's better. Or perhaps this lends to the question whether KilnApi should be broken up into two separate APIs. Or perhaps I can make the accountId a parameter for each method. Which seems best?

Reply:
It's only used in adaStakeTransaction just pass it in there